src/initialization/InitDistribution.cpp

```diff
@@ -32,6 +33,65 @@
 namespace impactx
 {
 
+    /** Ignore the shape of a distribution and use the 2nd moments to create a covariance matrix
+     */
+    CovarianceMatrix
+    create_covariance_matrix (
+        distribution::KnownDistributions const & distr
+    )
+    {
+        // zero out the 6x6 matrix
+        CovarianceMatrix cv;
+        for (int i=1; i<=6; ++i) {
+            for (int j = 1; j <= 6; ++j) {
+                cv(i, j) = 0.0;
+            }
+        }
```

Comment:
Memo to Axel: default is F-order of `amrex::Array2`. Double check loop indices are effective.